### PR TITLE
Develop web frontend draft

### PR DESCRIPTION
**Pull request by dexterastin**
I developed a mobile screen using google map api, card and carousel in materialize.

### CODE REVIEW

File: front-end/js/googleMap.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function initMap() {
+	var map = new google.maps.Map(document.getElementById('map'), {
+		center: {lat: -34.397, lng: 150.644},
```
Comment (khc196) [inline front-end/js/googleMap.js]: Where does the `center` variable point?

Comment (dexterastin) [reply front-end/js/googleMap.js]: Currently, I put a random value.

**Comment by khc196** [comment]
I don't know well from only looking code 😥

**Comment by dexterastin** [comment]
## pc preview
image

## mobile preivew
image

**Comment by khc196** [comment]
It looks great!

**Comment by mjkoo711** [comment]
Wow,,, the draft is good!
Thanks for your work!

**Comment by mjkoo711** [comment]
But I have a question.
What is the card interval in PC preview?

**Comment by dexterastin** [comment]
Padding of item is 10px. And shift is 10px.

**Comment by KooMinJun** [comment]
Um,,, I think that focus card is bigger than others.

File: front-end/css/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+/* Optional: Makes the sample page fill the window. */
+html, body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+.carousel {
+    position: absolute;
+    display: block;
```
Comment (shberry) [inline front-end/css/stylesheet.css]: Usually it's easier to read if you order css properties in lexicographical order :) (Or alphabetical, in this case)

File: front-end/css/stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+@media (min-width: 600px) {
+    .carousel {
+        height: 25vh;
+    }
+}
+
+.carousel .carousel-item {
+    opacity: 1 !important;
+}
```
Comment (shberry) [inline front-end/css/stylesheet.css]: I think we need a newline at the end of the file

File: front-end/index.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!--Import materialize.css-->
+    <link type="text/css" rel="stylesheet" href="css/materialize.min.css" media="screen,projection"/>
+    <link type="text/css" rel="stylesheet" href="css/stylesheet.css" media="screen,projection"/>
+    <!--Let browser know website is optimized for mobile-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+</head>
+
+<body>
+
```
Comment (shberry) [inline front-end/index.html]: remove extra line?

File: front-end/index.html [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+<div id="map"></div>
+
```
Comment (shberry) [inline front-end/index.html]: Too many extra lines

File: front-end/index.html [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+    <!--Import materialize.css-->
+    <link type="text/css" rel="stylesheet" href="css/materialize.min.css" media="screen,projection"/>
+    <link type="text/css" rel="stylesheet" href="css/stylesheet.css" media="screen,projection"/>
+    <!--Let browser know website is optimized for mobile-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+</head>
+
+<body>
+
+<div id="map"></div>
```
Comment (shberry) [inline front-end/index.html]: Maybe just <div id="map"/> ?

File: front-end/js/script.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+            label: labels[i % labels.length]
+        });
+    });
+    // Add a marker clusterer to manage the markers.
+    var markerCluster = new MarkerClusterer(map, markers,
+        {imagePath: 'https://developers.google.com/maps/documentation/javascript/examples/markerclusterer/m'});
+
+    for (i = 0; i < markers.length; i++) {
+        console.log(markers[i]);
+
```
Comment (shberry) [inline front-end/js/script.js]: Remove extra line

**Review by shberry — COMMENTED** [review]
Looks good overall :)

**Comment by dexterastin** [comment]
@shberry ah! thank you. 
I have modified the code.

**Pull request merged**